src/Kiota.Builder/Plugins/PluginsGenerationService.cs

```diff
@@ -613,7 +613,7 @@ private static Auth GetAuthFromSecuritySchemeReference(OpenApiSecuritySchemeRefe
             },
             SecuritySchemeType.OAuth2 => new OAuthPluginVault
             {
-                ReferenceId = string.IsNullOrEmpty(authenticationReferenceId) ? $"{{{name}_CONFIGURATION_ID}}" : authenticationReferenceId
+                ReferenceId = string.IsNullOrEmpty(authenticationReferenceId) ? $"{{{name}_REGISTRATION_ID}}" : authenticationReferenceId
```

Comment:
Should we align on casing also? TTK expects all upper case. Not sure if it's case sensitive though.

Reply:
Need to check for further clarification

Reply:
verified, this is what is expected on TTK side